config/ampache.cfg.php.dist

```diff
@@ -597,6 +597,18 @@ sociable = "true"
 ; DEFAULT: db,tags,folder,spotify,musicbrainz,lastfm,google
 art_order = "db,tags,folder,spotify,musicbrainz,lastfm,google"
 
+; Spotify Album art search filter
+;  Narrow the search. 
+;  POSSIBLE VALUES:  artist,(year:1991 or year:1991-2000)
+; POSSIBLE  VALUES: empty string("") or commented out for no filter
+; DEFAULT: none;
+;spotify_art_filter="artist"
+
+; Art search limit
+;  Limit the total images returned
+;  DEFAULT: 5
+art_search_limit="15"
```

Comment:
Was it your intention to set the default in the dist-config to 15? I guess, there's a `;` missing? Or did I get this wrong?

Reply:
Thanks, no, it wasn't. When I copied it over to the .dist file, I forgot to comment it.  Thanks for the heads up.  Taking care of it now.